tensorflow/core/kernels/mkl_fused_batch_norm_op.cc

```diff
@@ -292,14 +399,23 @@ struct MklBatchNormBwdParams {
         eps(eps),
         training(training),
         src_format(src_format) {}
+#else
+  MklBatchNormBwdParams(memory::dims src_dims, memory::dims diff_dst_dims,
+                        int depth, float eps, bool training)
+      : src_dims(src_dims),
+        diff_dst_dims(diff_dst_dims),
+        depth(depth),
+        eps(eps),
+        training(training) {}
+#endif
```

Comment:
Nit: (since this is pretty far from the `#ifdef`).
```suggestion
#endif  // !ENABLE_MKLDNN_V1
```

Reply:
Added for all